osm2gtfs/core/cache.py

```diff
@@ -37,3 +37,30 @@ def read_data(name):
             return content
         else:
             return {}
+
+    @staticmethod
+    def write_file(name, content):
+        """Function to write cache
+
+        Writes an object (content) with an indicated name to a file on the
+        hard drive.
+
+        """
+        if not os.path.isdir('data'):  # checking is the dir exist
+            os.mkdir('data')
+        with open('data/' + name, 'wb') as f:
+            f.write(content.read())
+
+    @staticmethod
+    def read_file(name):
+        """Function to read cache
+
+        Reads and returns an object (content) with an indicated name from a
+        file on the hard drive.
+
+        """
+        if os.path.isfile('data/' + name):
+            with open('data/' + name, 'rb') as f:
```

Comment:
While that works on Linux, it is better to use `os.path.join()` here. You can also introduce a variable for the path, since you use it twice.

Reply:
Yes, thanks, done in the whole class.